packages/edit-site/src/components/editor/index.js

```diff
@@ -217,9 +218,20 @@ export default function EditSiteEditor() {
 									<Button
 										label={ __( 'Open Navigation' ) }
 										className="edit-site-layout__view-mode-toggle"
-										onClick={ () =>
-											setCanvasMode( 'view' )
-										}
+										onClick={ () => {
+											setCanvasMode( 'view' );
+											// TODO: this is a temporary solution to navigate to the posts list if we are
+											// come here throught `posts list` and are in focus mode editing a template or template part.
+											if (
+												isPostsList &&
```

Comment:
I think that's fine but we should be careful about not using this variable in a lot of places.